x-pack/plugins/maps/public/routing/routes/list/maps_list_view.tsx

```diff
@@ -44,9 +52,30 @@ function navigateToNewMap() {
     path: MAP_PATH,
   });
 }
+interface MapsListViewState {
+  sortField?: string | number | symbol;
+  sortDirection?: Direction;
+  hasInitialFetchReturned: boolean;
+  isFetchingItems: boolean;
+  showDeleteModal: boolean;
+  showLimitError: boolean;
+  filter: string;
+  items: [];
+  selectedIds: [];
+  page: number;
+  perPage: number;
+  readOnly: any;
+  listingLimit: any;
+  totalItems?: number;
+}
+
+interface SelectionItem {
+  id: string;
+}
 
 export class MapsListView extends React.Component {
-  state = {
+  _isMounted?: boolean;
```

Comment:
Just default to false instead of typing as `?: boolean`

Reply:
Fixed this.